src/Sylius/Bundle/ReviewBundle/Resources/public/js/sylius-review.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$(document).ready(function(){
+    $("#reviewForm").submit(function(e){
+        e.preventDefault();
+        var form = $(this);
+
+        $.ajax({
+            type: "POST",
+            url: location.href+"/product_review",
```

Comment:
Urls should not use ``_``. Anyway, correct url for creating a new review would be:

POST ``/products/{productId}/reviews/``, not POST ``/product_review``.